datahub-web-react/src/appConfigContext.tsx

```diff
@@ -53,6 +53,7 @@ export const DEFAULT_APP_CONFIG = {
         nestedDomainsEnabled: true,
         platformBrowseV2: false,
         businessAttributeEntityEnabled: false,
+        dataContractsEnabled: true,
```

Comment:
I think there's one more place we need to add this flag to fully wire the mapping. I think it's `application.yaml` but I need to confirm - remind me to walk through this w you on our next standup pls

Reply:
Added